src/arch/host/lib/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof notifier.c)
+add_local_sources(sof notifier.c timer.c)
```

Comment:
@lgirdwood @lyakh  I noticed there is src/drivers/host/timer.c. Would it be better location for this dummy function? the others look like dummies too. But there's a #ifndef __ZEPHYR__ so it indicates it might be used with Zephyr.

Reply:
I did this change, hope it's OK.